en/report_infringement.md

```diff
@@ -21,7 +21,8 @@ The quickest way is to email it to [dmcanotice@mozilla.com](mailto:dmcanotice@mo
 
 Mozilla<br>
 Attn: Mozilla’s Designated DMCA Agent<br>
-2 Harrison St. #175<br>
+149 New Montgomery St. 
```

Comment:
The alternative to use `<br>` is to use two blank spaces at the end of the line. If you only put one, there will be no line break.